### ISSUE TITLE
Indicat language while you are learning

### ISSUE THREAD

**Issue by szabgab** [kantord/LibreLingo#1860]
The could be a flag or some icon in view representing the language we are currently learning on each screen while we are in one of the courses.

**Comment by kantord** [comment]
this is a very good idea but I'm not sure what would be the best way to implement it? To my knowledge flags are not a great way to represent languages. But I can't think of a better alternative other than simply putting the entire language name. But that can be inconvenient and it might now even fit, especially if the language name is very long.

Or perhaps we can associate languages with colors and show a color with the abbreviation of the language?

- 🔴 ZH
- 🔵 EN
- 🟢 PT
- 🟡 ES

not sure how well this would work 🤔 

I guess the main purpose of indicating the language is that people who are learning multiple languages at the same time can see which one they're learning at the moment? 🤔

shouldn't there also be a way to switch courses?

**Comment by szabgab** [comment]
Yes, the main purpose is for people learning multiple languages. Especially of they are similar. (e.g. Spanish and Ladino)

I agree, one should be able to switch courses :-).  That should be only while not in a learning session, right?

**Comment by kantord** [comment]
> That should be only while not in a learning session, right?

yes. in a learning session they would lose their progress. so better to make them explicitly quit the session first
